src/main/kotlin/racingcar/domain/IOHandler.kt

```diff
@@ -0,0 +1,72 @@
+package racingcar.domain
+
+import camp.nextstep.edu.missionutils.Console
+import racingcar.data.Car
+
+class IOHandler(private val validator: Validator) {
+    fun getCarNames(): List<String> {
+        val input = getInput()
+        val names = splitNameFromInput(input)
+
+        for (name in names) {
+            require(validator.checkCarName(name)) {
+                NAME_LENGTH_IS_NOT_MATCHED
+            }
+        }
+
+        return names
+    }
+
+    fun getMoveCount(): Int {
+        val input = getInput()
+        val count = changeNumStringToInt(input)
+
+        require(validator.checkMoveCount(count)) {
+            NOT_ALLOWED_COUNT
+        }
+
+        return count
+    }
+
+    fun showLocation(cars: List<Car>) {
+        for (car in cars) {
+            show(car)
+        }
+    }
+
+    fun showWinner(winners: List<Car>) {
+        val sentence = winners.joinToString(SEPARATOR_FOR_WINNER) {
+            it.name
+        }
+        show("$SENTENCE_FOR_WINNER$sentence")
+    }
+
+    fun show(sentence: String, lineBreak: Boolean = false) {
+        if (lineBreak) {
+            println(sentence)
+            return
+        }
+        print(sentence)
```

Comment:
[3] lineBreak를 통해서 출력 제어를 한 부분이 아이디어가 좋은 것 같네요.
취향이나 가독성 기준의 차이일 것 같은데, 아래 처럼 바꿀 수도 있을 것 같아서 남겨봅니다!!
```suggestion
        print(sentence)
        
        if (lineBreak) {
            println()
        }
```